Makefile

```diff
@@ -145,7 +145,7 @@ override undefine GOBIN
 # This must never include the 'hack' directory
 export PATH := $(shell $(GO) env GOPATH)/bin:$(PATH)
 
-GOMD2MAN ?= $(shell command -v go-md2man || echo './test/tools/build/go-md2man')
+GOMD2MAN ?= './test/tools/build/go-md2man'
```

Comment:
I am pretty sure you have to remove the single quotes as they are part of the variable